### PR TITLE
Leaderboard: Hit the api

### PR DESCRIPTION
**Pull request by JuanDiegoGuerra**
### Leaderboard: Hit the api ( 2nd milestone )
 - Followed the project requirements and completed the 2nd Milestone of **"Leaderboard Project"** ✅

Preview : 🕵🏼‍♂️

leaderboard_day3_preview

### CODE REVIEW

File: src/modules/fetchingApi.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const apiUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/lyyHMBWzgAOEuOOl3eGh/scores/';
+
+export const getScores = async () => {
+  const response = await fetch(apiUrl);
+  const jsonData = await response.json();
+  return jsonData.result;
+};
+
+export const addScore = async (user, score) => {
```
Comment (Ismailco) [inline src/modules/fetchingApi.js:9]: - [ ] When I try to send a new score I got a network error and no indication in the front end of anything happened, kindly make sure to fix the issue or add instructions and validation if I need to do something else before I send the score to the board

Screenshot_20230519_065336

**Review by Ismailco — APPROVED** [review]

File: src/modules/fetchingApi.js (hunk 1 above)
Comment (Ismailco) [inline src/modules/fetchingApi.js:9]: - [x] When I try to send a new score I got a network error and no indication in the front end of anything happened, kindly make sure to fix the issue or add instructions and validation if I need to do something else before I send the score to the board

Screenshot_20230519_065336

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @JuanDiegoGuerra, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/fetchingApi.js (hunk 1 above)
Comment (JuanDiegoGuerra) [reply src/modules/fetchingApi.js:9]: Yes! You are right, I didn't realize I changed my event listener. I just fixed it and will send a new review. Big thanks for pointing me the issue I had, I really apreacciate it!

**Comment by JuanDiegoGuerra** [comment]
###  Required changes made 🛠
**All linters are enable!**
- Fixed _"When I try to send a new score I got a network error and no indication in the front end of anything happened, kindly make sure to fix the issue"_ ✅
  - Now the app is adding an score with its user name correctly!

todolist1

File: package.json [hunk 2]
```diff
@@ -15,13 +15,13 @@
...
-    "css-loader": "^6.7.3",
+    "css-loader": "^6.7.4",
     "eslint": "^7.32.0",
     "eslint-config-airbnb-base": "^14.2.1",
     "eslint-plugin-import": "^2.27.5",
     "hint": "^7.1.8",
     "html-webpack-plugin": "^5.5.1",
-    "style-loader": "^3.3.2",
+    "style-loader": "^3.3.3",
     "stylelint": "^13.13.1",
```
Comment (omar-muhamad) [inline package.json:25]: - [x] [Optional] [Highly recommended] Please add `webpack-dev-server` to the devDependencies and also add `webpack`, `webpack-cli`, and `webpack-dev-server` to the regular dependencies by installing them like that:
```terminal
npm i webpack webpack-cli webpack-dev-server --save-dev 
```

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @JuanDiegoGuerra,
#### Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 
### ⚠️Please consider the Optional suggestions below⚠️

robert-redford-nod

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- Please check the inline comments below ⬇.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: package.json (hunk 2 above)
Comment (JuanDiegoGuerra) [reply package.json:25]: Thanks for the feedback! I will try to improve that.

**Review by AbelG101 — COMMENTED** [review]
# Hi @JuanDiegoGuerra 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- POST API endpoint is hit upon submission ✔
- GET API endpoint is hit upon refresh ✔
- Proper Use of async, await, and the fetch method ✔
- Correct Git flow✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**